### PR TITLE
added checkoutStepTwo Page and smokeTest tests files

### PR DESCRIPTION
**Pull request by LiviuAC**

### CODE REVIEW

File: test/pageobjects/CheckoutPage.js [hunk 1]
```diff
@@ -23,22 +27,94 @@ class CheckoutPage extends PageObject {
...
+
+    async checkoutProductData(data) {
+        switch(data){
+            case 'names':
+                let productsNamesText = []
+                const productNames = await this.allItemsName;
+                for (let i = 0; i < productNames.length; i++) {
+                    productsNamesText.push(await productNames[i].getText())
+                }
+                return productsNamesText
```
Comment (axboory) [inline test/pageobjects/CheckoutPage.js]: This code section is used multiple times in CheckoutPage and ProductsPage. Create a method for this with the elements as parameters.

File: test/pageobjects/CheckoutPage.js [hunk 2]
```diff
@@ -23,22 +27,94 @@ class CheckoutPage extends PageObject {
...
+                }
+                return productsPrices
+            case 'descriptions':
+                let productsDetailsText = [];
+                const productDetail = await this.allItemsDescriptions;
+                for (let i = 0; i < productDetail.length; i++) {
+                    productsDetailsText.push(await productDetail[i].getText());
+                }
+                return productsDetailsText
+        }
```
Comment (axboory) [inline test/pageobjects/CheckoutPage.js]: same as above

File: test/pageobjects/CheckoutPage.js [hunk 3]
```diff
@@ -23,22 +27,94 @@ class CheckoutPage extends PageObject {
...
+                return productsNamesText
+            case 'prices':
+                let productsPrices = [];
+                const productPrice = await this.allItemsPrices;
+                let numberPrice
+                for (let i = 0; i < productPrice.length; i++) {
+                    numberPrice = await productPrice[i].getText()
+                    productsPrices.push(numberPrice.replace("$", ""));
+                }
+                return productsPrices
```
Comment (axboory) [inline test/pageobjects/CheckoutPage.js]: same as above

File: test/specs/checkoutStepTwoPage.js [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+
+        it("check Shipping Information UI elements", async () => {
+
+            expect(await CheckoutPage.summaryInfoText.getText()).toContain("Shipping Information:");
+            expect(await CheckoutPage.summaryInfoText.getText()).toContain("FREE PONY EXPRESS DELIVERY!");
+        })
+
+        it("check Item total, Tax and Total Information UI elements", async () => {
+
+            console.log("info: ", await CheckoutPage.summaryInfoText.getText())
```
Comment (axboory) [inline test/specs/checkoutStepTwoPage.js]: remove if not needed

**Review by axboory — CHANGES_REQUESTED** [review]

File: test/pageobjects/CheckoutPage.js (hunk 1 above)
Comment (LiviuAC) [reply test/pageobjects/CheckoutPage.js]: a new method "extractTextData" has been created inside the PageObject file.

File: test/pageobjects/CheckoutPage.js (hunk 2 above)
Comment (LiviuAC) [reply test/pageobjects/CheckoutPage.js]: a new method "extractTextData" has been created inside the PageObject file.

File: test/pageobjects/CheckoutPage.js (hunk 3 above)
Comment (LiviuAC) [reply test/pageobjects/CheckoutPage.js]: a new method "extractTextData" has been created inside the PageObject file.

File: test/specs/checkoutStepTwoPage.js (hunk 4 above)
Comment (LiviuAC) [reply test/specs/checkoutStepTwoPage.js]: removed

File: test/pageobjects/CheckoutPage.js [hunk 5]
```diff
@@ -23,22 +25,66 @@ class CheckoutPage extends PageObject {
...
+            case 'oneItem':
+                await ProductsPage.btnAddBackpack.click()
+                break
+            case 'multipleItems':
+                await ProductsPage.addMultipleItems()
+                break
+        }
+        await ProductsPage.shoppingCartIcon.click()
+        await CartPage.btnCheckout.click()
+    }
```
Comment (axboory) [inline test/pageobjects/CheckoutPage.js]: shouldn't this method be in ProductsPage.js?

**Review by axboory — CHANGES_REQUESTED** [review]

File: test/pageobjects/CheckoutPage.js (hunk 5 above)
Comment (LiviuAC) [reply test/pageobjects/CheckoutPage.js]: the method was needed while doing the checkout step 2 tests, that's why was inside CheckoutPage.js 

it has been moved inside ProductsPage.js and the specs smokeTest and checkoutStepTwoPage have been updated.

**Review by axboory — APPROVED** [review]

**Pull request merged**